### ISSUE TITLE
php-cs-fixer not work in windows 8-64 

### ISSUE THREAD

**Issue by usitnikov** [Glavin001/atom-beautify#293]
Hi all!
I try to use atom-beautify on windows 8-64.
And i catch this error: 
Command failed: C:\WINDOWS\system32\cmd.exe /s /c "php ("C:\\ProgramData\\PHP Coding Standards Fixer\\bin\\php-cs-fixer.phar" fix "C:\Users\user\AppData\Local\Temp\input115330-7320-qu2bz8") & move "C:\Users\user\AppData\Local\Temp\input115330-7320-qu2bz8" "C:\Users\user\AppData\Local\Temp\115330-7320-1ctpyu3"" �� 㤠���� ����� 㪠������ 䠩�.

in console i viewed this:
C:\Users\user>C:\WINDOWS\system32\cmd.exe /s /c "php ("C:\\ProgramData\\PHP Coding Standards Fixer\\bin\\php-cs-fixer.phar" fix "C:\Users\user\AppData\Local\Tem
p\input115330-7320-qu2bz8")"
Could not open input file: (C:\\ProgramData\\PHP Coding Standards Fixer\\bin\\ph
p-cs-fixer.phar

I think this error was because in cli execution line after [ "php ] locate [ ( ].

Debug:
# Atom Beautify - Debugging information

The following debugging information was generated by `Atom Beautify` on `Thu Apr 30 2015 11:10:49 GMT+0500 (RTZ 4 (зима))`.

---

**Platform**: win32

## Versions

**Atom Version**: 0.194.0

**Atom Beautify Version**: 0.24.1

## Original file to be beautified

**Original File Path**: `Z:\html\module\Application\src\Application\Controller\AccountController.php`

**Original File Grammar**: PHP

**Original File Contents**: 
```PHP
<?php

namespace Application\Controller;

use Zend\Mvc\Controller\AbstractActionController,
    Zend\View\Model\ViewModel,
    Zend\View\Model\JsonModel;

class AccountController extends AbstractActionController {

    protected $ORM;

    public function __construct() {

    }

    public function indexAction() {
        $this->redirect()->toRoute('account/user');
    }

    public function userAction() {
        // echo $user_id = $this->zfcUserAuthentication()->getIdentity()->getId();

        $code = 'BB0102A';
        $search_surface = $this->getORM()->getRepository('\Application\Entity\outdoorSurfaces')->findOneBy([
          'owners_code' => $code,
          'client' => $user_id = $this->zfcUserAuthentication()->getIdentity()->getClient()->getId(),
          ]);

        echo $search_surface->getId();

        $this->layout()->setTemplate('layout/my_layout');
        return new ViewModel();
    }
}

```

## Beautification options

**Editor Options**: 
Options from Atom Editor settings
```json
{
    "indent_size": 2,
    "indent_char": " ",
    "indent_with_tabs": false
}
```

**Config Options**: 
Options from Atom Beautify package settings
```json
{
    "php": {
        "cs_fixer_path": "C:\\ProgramData\\PHP Coding Standards Fixer\\bin\\php-cs-fixer.phar",
        "fixers": "",
        "level": ""
    },
    "js": {
        "indent_size": 2,
        "indent_char": " ",
        "indent_level": 0,
        "indent_with_tabs": false,
        "preserve_newlines": true,
        "max_preserve_newlines": 10,
        "space_in_paren": false,
        "jslint_happy": false,
        "space_after_anon_function": false,
        "brace_style": "collapse",
        "break_chained_methods": false,
        "keep_array_indentation": false,
        "keep_function_indentation": false,
        "space_before_conditional": true,
        "eval_code": false,
        "unescape_strings": false,
        "wrap_line_length": 0,
        "end_with_newline": false
    },
    "css": {
        "indent_size": 2,
        "indent_char": " ",
        "selector_separator_newline": false,
        "newline_between_rules": false,
        "preserve_newlines": false
    },
    "html": {
        "htmlbeautifier_path": "",
        "indent_inner_html": false,
        "indent_size": 2,
        "indent_char": " ",
        "brace_style": "collapse",
        "indent_scripts": "normal",
        "wrap_line_length": 250,
        "wrap_attributes": "auto",
        "wrap_attributes_indent_size": 2,
        "preserve_newlines": true,
        "max_preserve_newlines": 10,
        "unformatted": [
            "a",
            "sub",
            "sup",
            "b",
            "i",
            "u"
        ],
        "end_with_newline": false
    },
    "sql": {
        "indent_size": 2,
        "keywords": "upper",
        "identifiers": "lower",
        "sqlformat_path": ""
    },
    "markdown": {
        "pandoc_path": "",
        "yaml_front_matter": true
    },
    "perl": {
        "perltidy_path": "perltidy",
        "perltidy_profile": ""
    },
    "python": {
        "autopep8_path": "",
        "max_line_length": 79,
        "indent_size": 2,
        "ignore": [
            "E24"
        ]
    },
    "ruby": {
        "rbeautify_path": ""
    },
    "c": {
        "uncrustifyPath": "",
        "configPath": ""
    },
    "cpp": {
        "uncrustifyPath": "",
        "configPath": ""
    },
    "objectivec": {
        "uncrustifyPath": "",
        "configPath": ""
    },
    "cs": {
        "uncrustifyPath": "",
        "configPath": ""
    },
    "d": {
        "uncrustifyPath": "",
        "configPath": ""
    },
    "java": {
        "uncrustifyPath": "",
        "configPath": ""
    },
    "pawn": {
        "uncrustifyPath": "",
        "configPath": ""
    },
    "vala": {
        "uncrustifyPath": "",
        "configPath": ""
    }
}
```

**Home Options**: 
Options from `C:\Users\user\.jsbeautifyrc`
```json
{}
```

**EditorConfig Options**: 
Options from [EditorConfig](http://editorconfig.org/) file
```json
{}
```

**Project Options**: 
Options from `.jsbeautifyrc` files starting from directory `Z:\html\module\Application\src\Application\Controller` and going up to root
```json
[
    {},
    {},
    {},
    {},
    {},
    {}
]
```

## Logs

**Error logs**: *Not yet supported*

**Comment by Glavin001** [comment]
I just published a new release to v0.25.0

This is a very big release with lots of internal changes and (hopefully) no breaking changes. Please let me know if you are experiencing any unusual behaviour / breaking changes after updating. Thank you.

This may even resolve your issue. If not, now we can continue debugging from here and the internal code for Atom Beautify should be much easier to debug :+1:.

**Comment by Glavin001** [comment]
> I think this error was because in cli execution line after [ "php ] locate [ ( ].

You are correct. I noticed this typo earlier this week and fixed it. So you should be working now with v0.25.0 and up.
